src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -1072,11 +1072,12 @@ To use a custom JDK path for a command line build, set the 'JavaSdkDirectory' MS
 {1} - NuGet package version</comment>
   </data>
   <data name="XA0141" xml:space="preserve">
-    <value>NuGet package '{0}' version '{1}' contains a shared library '{2}' which is not correctly aligned. See https://developer.android.com/guide/practices/page-sizes for more details</value>
+    <value>'{2}' from NuGet package '{0}' version '{1}' contains a shared library '{3}' which is not correctly aligned. You may wish to inform the creators of the Library about this issue. See https://developer.android.com/guide/practices/page-sizes for more details.</value>
```

Comment:
@jonpryor can you remember me how we wanted to change this message? I forgot to take notes.